src/code/graph.c

```diff
@@ -377,7 +402,10 @@ void Graph_Update(GraphicsContext* gfxCtx, GameState* gameState) {
 
     {
         OSTime timeNow = osGetTime();
-        s32 pad[4];
+        s32 pad;
+#if OOT_DEBUG
+        s32 pad2[3];
```

Comment:
Could these padding variables go somewhere in the `#if OOT_DEBUG` just below instead of here?

Reply:
Yes, they actually are probably part of the SET_NEXT_GAMESTATE macro, so I did what was done in z_select.c